src/main/webapp/feed.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Message Feed</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="stylesheet" href="/css/user-page.css">
+
+  <script>
```

Comment:
Consider moving this script to separate file under js folder. 
There are other functionalities also require the similar script. So all can share the same code. For now, just create a new file and use it from here.

Reply:
I will move this script under the js folder on the next pull request. Thank you for the suggestion!